views/js/qtiCommonRenderer/tpl/interactions/extendedTextInteraction.tpl

```diff
@@ -52,4 +52,5 @@
             </div>
         {{/if}}
     {{/if}}
+  <ins class="interaction-cover"></ins>
```

Comment:
since this bug is affecting the interaction authoring only, I would suggest not affecting the runtime code/template but only the authoring code. 
2 possibilities: 
1. overriding the template in the creator renderer of the extended text interaction
2. append the element (from a template) in the widget intiialization

I tend to find the first solution cleaner.

Reply:
I now did it without any element on the `::after` of the widget. I think that's even nicer.